src/app/components/projects/site-card/site-card.component.ts

```diff
@@ -41,10 +41,21 @@ import { Site } from "@models/Site";
             </a>
           </li>
           <li *ngIf="site" class="nav-item">
-            <a id="play" class="nav-link" [routerLink]="site.playUrl">
+            <!-- Play link if recording exists -->
+            <a
+              *ngIf="recording"
+              id="play"
+              class="nav-link"
+              [routerLink]="recording?.viewUrl"
+            >
```

Comment:
Should fallback to "No audio yet" if no audio link available. Maybe with a disabled play icon still?